pytripgui/controller/main_cont.py

```diff
@@ -100,7 +98,7 @@ def open_dicom_dialog(self, event):
             return
         self.open_dicom(ddir)
         model.dicom_path = ddir
-        st.save("general.import.dicom_path", ddir)
+        st.save()
```

Comment:
There are too many unnecessary assignments, calling simply `self.settings.save()` would make the code more readable.

Reply:
Note regarding assignments:
I use excessive assignments during development, because readability is a trade-of situation.

If a variable name is used many times and it has a long filename, assigning it to a shorter name increases readability.
If a variable name is used very few times (such as in this case), it is the other way round.

Since this is in development, I often cannot foresee just yet how often I am going to use some attribute names, so the assignments are left in, and may be refactored once the codebase gets stable.

(Furthermore it simplifies manual refactoring during development.)